docs-ref-services/storage.md

```diff
@@ -16,19 +15,79 @@ The Azure Storage libraries for Java provide classes for working with data in yo
 
 ## Client library for data access
 
-Use the Azure Storage client library for Java to work with data in your storage account. The following table describes the recommended versions of the storage client library for Java.
+Use the Azure Storage client library for Java to work with data in your storage account:
+
+- Read and write objects and files from [Azure blob storage](/azure/storage/blobs/storage-blobs-introduction)
```

Comment:
Azure Blob is a service name, so we capitalize the word Blob in the link: \[Azure Blob storage\]